jetbrains-ultimate/resources-201/META-INF/ext-datagrip.xml

```diff
@@ -0,0 +1,12 @@
+<!-- Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved. -->
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+
+<idea-plugin>
+    <extensions defaultExtensionNs="aws.toolkit">
+        <explorer.serviceNode implementation="software.aws.toolkits.jetbrains.core.explorer.nodes.RdsExplorerRootNode"/>
+    </extensions>
+    <extensions defaultExtensionNs="com.intellij.database">
+        <!-- order="first" comes from how other instances of connectionInterceptor are implemented in DataGrip -->
```

Comment:
What is causing this to be first? What behavior are we seeing?

Reply:
It doesn't load if we don't put it as first, at least when I tried it. Looked at some other examples and they seem to have order first

Reply:
Order doesnt matter when everyone says first 😛

Reply:
Was able to remove it in the branch with functionality, removing this entirely from this PR